core/schema_utils.py

```diff
@@ -682,3 +682,28 @@ def is_valid_username_string(obj: str) -> bool:
             return True
         except utils.ValidationError:
             return False
+
+    @staticmethod
+    def should_end_with(
+        obj: str,
+        choices: List[str],
+        match_case: Optional[bool] = False
+    ) -> bool:
+        """Checks if the given obj (a string) is a filename with
+        valid audio extension.
+
+        Args:
+            obj: str. The string to verify.
+            choices: List[str]. All valid extensions.
+            match_case: bool. Whether the match is case sensitive.
+
+        Returns:
+            bool. Whether the given object is a filename with valid
+            audio extension.
+        """
+        dot_index = obj.rfind('.')
```

Comment:
Why `dot`? This function should verify whether the given object ends with any of the choices parameters, right?

Reply:
I assumed the function would be used to check for file extensions. I have removed this and used a more general approach.